src/app/pdp/directives/ot-social-links.html

```diff
@@ -0,0 +1,31 @@
+<!-- social links -->
+<div class="social">
+    <div id="social" class="clearfix">
+        <p class="share-this">Share this item:</p>
+
+        <div class="google-plus" ng-if="showGooglePlus">
+            <g:plusone size="medium" annotation="none"></g:plusone>
+        </div>
+
+        <div class="tweet-btn" ng-if="showTwitter">
+            <a href="http://twitter.com/share" class="twitter-share-button" data-url="http://knickknackrecords.com/products/a-ha-scoundrel-days-cassette" data-count="none" data-via="shopify">Tweet</a><script type="text/javascript" src="//platform.twitter.com/widgets.js"></script>
```

Comment:
@igorGalchenko data-url should contain current url, not knick-knack records product url